runners/google-cloud-dataflow-java/worker/windmill/src/main/proto/windmill.proto

```diff
@@ -784,6 +827,17 @@ message StreamingGetDataResponse {
   repeated bytes serialized_response = 2;
   // Remaining bytes field applies only to the last serialized_response
   optional int64 remaining_bytes_for_response = 3;
+
+  // Work item fingerprints available only for complete items. So if
+  // remaining_bytes_for_response > 0 the size of farmhash_fingerprint array is
+  // one less than the size of serialized_response as the fingerprint for the
+  // last message is not yet available.
+  // Will not be populated when heartbeat_response is set.
+  repeated uint64 farmhash_fingerprint = 4;
```

Comment:
can we remove this? since it is not used and is unrelated to the change.

Reply:
I copied this from the internal windmill.proto. I'm worried if I don't add it then the tags will be off. I guess I could mark 4 as reserved for now if you think that's better (but I think the heartbeat field needs to have tag 5).

Reply:
skipping 4 without reserving it should also work. I'm okay if you want to keep it.

Reply:
I'd say omit it, and either reserve or just skip the #.
Since it's matching the internal proto, I'm not really that worried someone will add a bad field # here.

Reply:
Sounds good. I'll just reserve 4 to be safe.